src/platform/test/functional/apps/console/_misc_console_behavior.ts

```diff
@@ -21,7 +21,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
   describe('misc console behavior', function testMiscConsoleBehavior() {
     before(async () => {
-      await browser.setWindowSize(1200, 800);
```

Comment:
This was removed because of [this failing test](https://buildkite.com/elastic/kibana-pull-request/builds/301068#0196ee9c-03a1-4c03-a30c-96b89f8b3c7e), strange thing is the expectation is met whilst running this PR locally as long the window isn't shrunk, however for FTR tests because of the window size, this test PR fails, my guess would be that some kind of anchor scroll with the toc messes with the URL.

I've linked a video showing the experience;

https://github.com/user-attachments/assets/303bb29f-112c-469e-92e7-01a6f5047286